_posts/2024-09-04-App-Service-Community-Standup.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Azure App Service Community Standup: .NET 9 and App Configuration"
+author_name: "Denver Brittain"
+toc: true
+toc_sticky: true
+---
+
+Watch the latest Azure App Service Community Standup on .NET 9 and App Configuration
+
+[![Azure App Service Community Standup](https://img.youtube.com/vi/Vj2ujuj5_Vo/hqdefault.jpg)](https://www.youtube.com/watch?v=Vj2ujuj5_Vo)
+
+For more information on .NET 9 and App Configuration on App Service, check out the following resources:
+
+* [Generally Available: App Configuration references on App Service](https://azure.microsoft.com/en-us/updates/v2/app-config-ref-ga)
```

Comment:
please remove the /en-us/ segment form the URL, this prevents localization redirections from doing the right thing....

For this example, the URL should be: https://azure.microsoft.com/updates/v2/app-config-ref-ga